app/pages/lab/workflow-components/task-picker.jsx

```diff
@@ -0,0 +1,49 @@
+import taskComponents from '../../../classifier/tasks';
+import TaskIcon from './task-icon.jsx';
+
+function TaskButton({ active, definition, firstTask, onClick, value }) {
+  const taskDefinition = taskComponents[definition.type]?.getTaskText(definition);
+  const classNames = ['secret-button', 'nav-list-item'];
+  if (active) {
+    classNames.push('active');
+  }
+  return (
+    <div>
+      <button
+        aria-pressed={ active ? 'true' : 'false' }
+        type="button"
+        className={classNames.join(' ')}
+        name="selectedTask"
+        value={value}
+        onClick={onClick}
+      >
+        <TaskIcon type={definition.type} />
+        {' '}
+        {taskDefinition || 'Task editor is unavailable'}
```

Comment:
Should the button be clickable when the label is "Task editor is unavailable."?

Reply:
I think it's perfectly OK for a Task with no Task Editor to be clickable.

_Screenshot: Planet Hunters TESSting with a Light Curve Task, which has no editor._
<img width="609" alt="image" src="https://user-images.githubusercontent.com/13952701/205034376-e38c87f9-0a33-4191-8c6e-ce4df0506a50.png">

(Note: some jiggering was required to display this; see full review notes.)